src/api/test/protectedApiClient.test.ts

```diff
@@ -0,0 +1,135 @@
+import nock from 'nock';
+import ProtectedApiClient, {
+  ProtectedApiClientRoutes,
+  AdminApiClientRoutes,
+} from '../protectedApiClient';
+
+const BASE_URL = 'http://localhost';
+
+// TODO: handle invalid cases
```

Comment:
Can you add a ticket for this on Monday on the bugs/tech debt section? Don't want this to get lost. Or finish it as part of this PR.